include/oneapi/dpl/pstl/hetero/dpcpp/execution_sycl_defs.h

```diff
@@ -276,6 +257,14 @@ struct __ref_or_copy_impl<execution::device_policy<PolicyParams...>, _T>
     using type = _T;
 };
 
+// Extension: hetero execution policy type trait
+template <typename... Args>
```

Comment:
I think, `Args` should instead be `PolicyParams` to match the others here.

Reply:
Fixed